dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_item_click_panel.tsx

```diff
@@ -19,7 +19,11 @@ export interface TitleProps {
 
 export const DataConfigItemClickPanel = ({ title, isSecondary, closeMenu }: TitleProps) => {
   const icon = isSecondary && (
-    <EuiIcon type="arrowLeft" onClick={closeMenu} data-test-subj="panelCloseBtn" />
+    <EuiIcon
+    style={{cursor:'pointer'}}
```

Comment:
styling issue here, can we run prettier?

Reply:
okay done